lib/Model/Attendee.php

```diff
@@ -95,10 +95,10 @@ class Attendee extends Entity {
 	/** @var int */
 	protected $roomId;
 
-	/** @var string */
+	/** @var null|string */
```

Comment:
this can not be null?

Reply:
lets keep this separate for now

Reply:
I identified and fixed the problem. Was a problem on tests that created Atendee entity without type.